buttons.py

```diff
@@ -7,7 +7,8 @@
 class Button:
     _instances = []
 
-    def __init__(self, name, xxx_todo_changeme, centre=True):  # eg ('plus',(30,40))
+    def __init__(self, name, xxx_todo_changeme, centre=True):

```

Comment:
There are placeholder variables in the argument list, please fix them :grin:

Reply:
Thanks for noticing! Fixed in `7a470b4`